src/web/index.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  formatArgument,
+  interceptConsoleLogs,
+  patchConsole,
+} from './interceptConsoleLogs';
+
+patchConsole();
```

Comment:
I changed my mind and think we should call patchConsole inside interceptConsoleLogs web https://github.com/qawolf/playwright-utils/pull/5/files#diff-4c9dfea888dc81bdec426542e0175627R50